backend/src/main/java/com/jobwait/spring/RequestNavigation.java

```diff
@@ -54,10 +60,12 @@ public static String getUserAnswers(@RequestParam("at") String authToken) {
 	public static ResponseEntity<String> submitUserAnswers(@RequestParam("at") String authToken,
 			@RequestBody String payload) {
 		try {
-			Answers answers = Utils.mapper.readValue(payload, Answers.class);
+			JsonNode root = Utils.mapper.readTree(payload).path("answers");
+			Answer[] answersArray =Utils.mapper.treeToValue(root,Answer[].class);
+			List<Answer> answersList = Arrays.asList(answersArray);
```

Comment:
no way to merge these two lines?

Reply:
Sure, I can merge them

Reply:
done dc00e0db5f51d1ff961629e50de5941610920327